control/setup_control.go

```diff
@@ -272,6 +274,7 @@ func (p *PIDLoop) basicControlConfig(endpointName string, pidVals PIDConfig, con
 					"kD":             pidVals.D,
 					"kI":             pidVals.I,
 					"kP":             pidVals.P,
+					"PIDSets":        []*PIDConfig{&pidVals},
```

Comment:
prob need to remove this since we didn't test your code on hardware. Or we can try testing, but I think this might interact negatively with other prs going on in the controls code.